roofit/roofit/inc/RooPyBindFunction.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Project: RooFit
+ * Authors:
+ *   Robin Syring, CERN 2024
+ *
+ * Copyright (c) 2023, CERN
+ *
+ * Redistribution and use in source and binary forms,
+ * with or without modification, are permitted according to the terms
+ * listed in LICENSE (http://roofit.sourceforge.net/license.txt)
+ */
+
+#ifndef ROOPYBINDFUNCTION_H
+#define ROOPYBINDFUNCTION_H
+
+#include "RooAbsReal.h"
+#include "RooListProxy.h"
+#include "RooArgList.h"
+
+class RooPyBindFunction : public RooAbsReal {
+public:
+   RooPyBindFunction(const char *name, const char *title, RooArgList &varlist);
+   RooPyBindFunction(const RooPyBindFunction &right, const char *name = nullptr);
+   virtual ~RooPyBindFunction();
+
+   RooPyBindFunction* clone(const char *name) const override;
+   Double_t evaluate() const override;
```

Comment:
```suggestion
   double evaluate() const override;
```
The legacy ROOT type `Double_t` is not nececarry anymore. You can always use `double`, `float`, and `bool` instead of `Double_t`, `Float_t`, and `Bool_t`.